projects/models.nix

```
@@ -96,6 +107,7 @@ rec {
         programs = optionalAttrs (option programType);
         services = optionalAttrs (option serviceType);
       };
+      blobs = option (list blobType);
```

According to [Wikipedia](https://en.wikipedia.org/wiki/Binary_blob):

> In the context of [free and open-source software](https://en.wikipedia.org/wiki/Free_and_open-source_software), [proprietary software](https://en.wikipedia.org/wiki/Proprietary_software) only available as a [binary executable](https://en.wikipedia.org/wiki/Executable) is referred to as a blob or binary blob.

But what are binaries coming from non-proprietary software? just binaries?

```suggestion
      binary = option (list binaryType);
```

---

Yes, just binaries